api/.env.example

```diff
@@ -102,3 +102,6 @@ FEATURE_NOTIFICATIONS=true
 
 # Laravel Scout Driver
 SCOUT_DRIVER=pgsql
+
+# Larave Storage
```

Comment:
```suggestion
# Laravel Storage
```

Reply:
Good catch. Updated in [fix logging](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9881/commits/9798e8538a7b90bbd101fea36bd9db9fec4d785d).